packages/gatsby/package.json

```
@@ -132,6 +136,7 @@
     "string-similarity": "^1.2.2",
     "style-loader": "^0.23.1",
     "terser-webpack-plugin": "^1.4.3",
+    "theme-ui": "^0.3.0",
```

We're using emotion in restartScreen directly so we should add it to the package.json
```suggestion
    "theme-ui": "^0.3.0",
    "@emotion/core": "^10.0.28"
```

We should make sure that we do not get conflicts in the user's code that also use `@emotion/core` or `theme-ui`. We have this issue with React and gatsby-cli when people install it with npm.

---

I'm solely using React + theme-ui as a templating language for the RestartingScreen—would be happy to move to a raw HTML & CSS if that's preferable for folks 👍

---

I think the issue is a bit long term - we prolly do want to have this in Gatsby Admin when we ship it. I think we should figure out a strategy on how we could include those packages without breaking this for users.

---

This is separate from Gatsby Admin (which I've also figured out though*), as the Restarting Screen is literally sent to the client as raw HTML & CSS (`res.end(renderToString())`). That means it shouldn't conflict with our users configurations, now that I think about it that way, right? 🤔 

**see https://github.com/gatsbyjs/gatsby/pull/23368 even though that isn't going to be merged the solution should work regardless!*

---

The issue is how npm resolves dependencies - it could install wrong version of emotion and theme-ui because we have them in dependencies and then user would get wrong version. We have that happening with React.

---

Since gatsby-plugin-admin builds the site during publishing, we should be able to move most of the dependencies to devDependencies

---

This is just something I've seen before with `gatsby-cli` this might not be the case for these packages so, give it a go with NPM and a different version of emotion core and see if correct versions are loaded.

---

I have "fixed" this for now by moving the restarting screen to raw html & css! I will take a look at this with admin once we actually want to ship that, added to the project board so I don't forget: https://github.com/gatsbyjs/gatsby/projects/22#card-37219543